proposals/0046-optimistic-cluster-restart-automation.md

```diff
@@ -0,0 +1,313 @@
+---
+simd: '0046'
+title: Optimistic cluster restart automation
+authors:
+  - Wen Xu (Solana Labs)
+category: Standard
+type: Core
+status: Draft
+created: 2023-04-07
+feature: (fill in with feature tracking issues once accepted)
+---
+
+## Summary
+
+During a cluster restart following an outage, use gossip to exchange local
```

Comment:
nit: "enter a separate recovery protocol that uses gossip to exchange"

Reply:
Changed